std/format.d

```diff
@@ -8448,7 +8448,7 @@ printFloat_done:
     import core.memory;
     auto stats = GC.stats;
 
-    char[256] buf;
+    char[512] buf;
```

Comment:
I changed this, because the call in `formatValueImpl` also provides a buffer with 512 chars.